### ISSUE TITLE
Game crashes after loading the map

### ISSUE THREAD

**Issue by MatzeAHG** [GTNewHorizons/GT-New-Horizons-Modpack#15002]
### Your GTNH Discord Username

matzeahg

### Your Pack Version

2.4.0 Java 17-20

### Your Server

SP

### Java Version

Java 20

### Type of Server

Single Player

### Your Actions

1-3 seconds after I load into my SP World the game crashes. I switched from 2.3.4 a week ago because I had the same problem on that version and I thought that the update fixed the issue because I didnt had any problems since then until today. I used to play with OptiFine and guessed that the crash is caused by that but even without OptiFine the games crashes. 

What I tried until now:

1. Removing OptiFine from mods folder after the game crashes with OptiFine (and identifies OptiFine as a potential cause)
2. Restarting PC

### Crash Report

[crash-2023-12-04_15.47.54-client.txt](https://github.com/GTNewHorizons/GT-New-Horizons-Modpack/files/13547564/crash-2023-12-04_15.47.54-client.txt)

### Final Checklist

- [X] I have searched this issue tracker and there is nothing similar already. Posting on a closed issue saying the crash still exists will prompt us to investigate and reopen it once we confirm your report.
- [X] I can reproduce this crash consistently by following the exact steps I described above.
- [ ] I have asked other people and they confirm they also crash by following the exact steps I described above.

**Comment by bombcar** [comment]
It’s crashing in the sound library which is a strange place to crash. Can you try a new world and see if it also explodes? The full fml-latest may help.

**Comment by MatzeAHG** [comment]
So I tried it again a few times and I guess part of the crash is that i didnt have a audio output connected. At least connecting headphones does look like a fix (strange that I didnt found this out earlier), but still without a audio output the crash does not happen everytime... just like 70%. 

I guess with fml latest you mean the fml-client-latest.txt?

**Comment by bombcar** [comment]
Yes, we shouldn't crash entirely without a sound output, so fml-client-latest may help figure out why.

(It's on J20 - I suspect if you tried the J8 version without audio you'd not get this)

**Comment by MatzeAHG** [comment]
Sure.

[fml-client-latest.log](https://github.com/GTNewHorizons/GT-New-Horizons-Modpack/files/13563188/fml-client-latest.log)

Also I tried making a new world but without headphones connected the game still crashes.

**Comment by eigenraven** [comment]
Most likely caused by <https://github.com/GTNewHorizons/lwjgl3ify/issues/100>
